gst-play/gst-play.c

```diff
@@ -195,9 +195,12 @@ static void
 print_all_stream_info (GstPlayerMediaInfo * media_info)
 {
   guint count = 0;
-  GList *l, *list;
+  GList *list, *l;
 
   list = gst_player_media_info_get_stream_list (media_info);
+  if (!list)
+    return;
```

Comment:
Even if we have no stream list here, we could at least print the URI, duration and other stream-list-independent things below.